src/PayPo/Request/Transaction/CancelRequest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\PayPo\Request\Transaction;
+
+use Answear\PayPo\Enum\OrderStatusEnum;
+
+class CancelRequest implements RequestInterface
+{
```

Comment:
I'he just saw that `\Answear\PayPo\Service\Order::reject` works similar. But PayPo documentation says that we can change to only CANCELLED OR CONFIRMED status.
Maybe you can remove reject flow from this bundle on separate PR?

Reply:
fyi https://github.com/answear/pay-po/pull/6